index.js

```diff
@@ -366,6 +366,11 @@ Tree.prototype._join = function (data, next) {
     , exit = _node.exit()
     , update = this.node = enter.merge(_node)
 
+  if (exit.size()) {
+    // Nodes are being removed, trigger an event for interested parties
+    this.emit('rebind:exit', exit)
```

Comment:
I wonder if an event name like `nodes-removed` or something would be more descriptive of the behavior? I think the terms `rebind` and `exit` may better describe the implementation than the behavior

Reply:
I struggled w/ the name. I want to keep the concept of `exit` because it's a d3 exit selection. The nodes aren't removed from the DOM at this point. I went with `rebind` because it's a specific form of rebinding the data.

I could settle on `exiting-nodes` if you prefer it.

Reply:
I think I don't fully understand what's going on here, so whichever name you like work best works for me